website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
 ---
-title: Getting Started
+title: Getting Started with a Local Talos Cluster 1.10
```

Comment:
I feel like the word "Local" might be misleading given that the guide is aimed at physical hardware. I think I'd be less confusing if the term was removed and you mentioned that this page is meant for physical machines on a local network somewhere in the beginning.